### PR TITLE
IfcDistributionPort Rail Import BIMQ 2021-07-21T06:37:58.2452980Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcDistributionPort

### CODE REVIEW

File: IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IK_Code_2kpG9_rK93LwNz4pCQcRnH" Name="IK_Code" UniqueId="aecd027e-d542-4357-a5fd-13331a99bc51" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IK_Code_2kpG9_rK93LwNz4pCQcRnH" Name="IK_Code" UniqueId="aecd027e-d542-4357-a5fd-13331a99bc51" PrimaryDataType="IfcLabel" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml:3]: Not used here.

File: IFC4x3/PropertyEnumerations/c/PEnum_CommunicationStandard.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		<DocPropertyConstant xsi:nil="true" href="STM_4_0hjW8nZin0LuzJLLFhPdKR" />
+		<DocPropertyConstant xsi:nil="true" href="STM_64_3dxRkcsFrFBepNhQhySrQO" />
+		<DocPropertyConstant xsi:nil="true" href="USB_34vpCdrEQgw44rcmQrmL" />
+		<DocPropertyConstant xsi:nil="true" href="XDSL_2YgIGrOFH5bhfztIYioqvK" />
+		<DocPropertyConstant xsi:nil="true" href="OTHER_0SAtxxJj15wQA15w2MsIV" />
+		<DocPropertyConstant xsi:nil="true" href="NOTKNOWN_0_n254Fb10hBW_0_XIj7Qe" />
+		<DocPropertyConstant xsi:nil="true" href="UNSET_01y959UgrAlvv6_W__S9Ap" />
+	</Constants>
+</DocPropertyEnumeration>
+
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/PropertyEnumerations/c/PEnum_CommunicationStandard.xml:17]: We didn't document enums, but values are quite self-explanatory.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_WiredCommunicationPortCommon/DocPropertySet.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_WiredCommunicationPortCommon" UniqueId="e2e65b77-4776-14a4-77a1-f0c9732ec743" ApplicableType="IfcDistributionPort/CABLE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="CommunicationStandard_23pIPVZq9A19s2K0Qc6rmz" />
+		<DocProperty xsi:nil="true" href="MaximumTransferRate_0Az4uYaFHDRAo465uiJHuc" />
+	</Properties>
+</DocPropertySet>
+
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_WiredCommunicationPortCommon/DocPropertySet.xml:8]: Done. Commit : b012c195b329226a7f7b2a0b0d5eec4660c87f99

**Review by larswik — APPROVED** [review]

File: IFC4x3/Properties/c/CommunicationStandard_23pIPVZq9A19s2K0Qc6rmz/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Indicate the communication standard supported by the physical wired communication port.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/c/CommunicationStandard_23pIPVZq9A19s2K0Qc6rmz/Documentation.md:1]:
```suggestion
Indicates the communication standard supported by the physical wired communication port.
```

File: IFC4x3/Properties/i/IsWelded_1ZxOPXMGT17eoWdZAFb$Ls/Documentation.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Indicate whether the wire pair end point is joined to another wire pair end point by means of a welded junction or not.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/i/IsWelded_1ZxOPXMGT17eoWdZAFb$Ls/Documentation.md:1]:
```suggestion
Indicates whether the wire pair end point is joined to another wire pair end point by means of a welded junction.
```

File: IFC4x3/Properties/m/MaximumTransferRate_0Az4uYaFHDRAo465uiJHuc/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Indicate the transmission rate in bit/s over the wired port.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MaximumTransferRate_0Az4uYaFHDRAo465uiJHuc/Documentation.md:1]:
```suggestion
Indicates the transmission rate in bit/s over the wired port.
```

**Review by SergejMuhic — COMMENTED** [review]
Almost there

**Pull request merged**